lib/stormpath-sdk/resource/application.rb

```diff
@@ -117,6 +118,10 @@ def get_provider_account request
   def oauth_authenticate(request)
     Stormpath::Oauth::Authenticator.new(data_store).authenticate(href, request) 
   end
+  
+  def create_login_attempt(options = {})
```

Comment:
Yes you are right there is already 
```ruby
has_many :login_attempts, can: :create
```
in the application.rb so we can just call
```ruby
application.login_attempts.create({
  type: “type”
  value: base64_encode("jsmith:mySecretPassword")
  accountStore: “accountStore or nameKey”
})
```
however what i wanted to achieve here is for the sdk to make the encoding and user just passes username and password to the method. there is a encode_credentials method in application which does this. This is just an opinion we can remove this if it is unnecessary.